src/logger.js

```diff
@@ -143,8 +163,9 @@ const error = (...args) => {
 
 const requestLog = (identifierMsg, { metadata, requestDetails: { url, body, method } }) => {
   const logger = getLogger();
-  if (LOGLEVELS[logLevel] === LOGLEVELS.warn) {
-    const reqLogArgs = [identifierMsg, { metadata, url, body, method }];
+  const filteredMetadata = getMatchedMetadata(metadata);
+  if (LOGLEVELS[logLevel] === LOGLEVELS.warn && filteredMetadata.length > 0) {
```

Comment:
why do we still need dependency on `warn` level ?
As long as destID/workspaceID are empty in env variables, it doesn't get logged.
Better not to disturb log level for these kind of changes - it has to be switched only from info --> debug or vice versa.

Reply:
Will remove the loglevel condition
Will use `info` instead of `warn` while logging. Should be fine right ?!

Reply:
Done with the changes